src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -421,23 +421,23 @@ public class ReleaseService : IReleaseService
 
         public async Task<Either<ActionResult, DeleteDataFilePlan>> GetDeleteDataFilePlan(Guid releaseId, Guid fileId)
         {
-            return await _persistenceHelper.CheckEntityExists<Release>(releaseId)
+            return await _context.Releases
+                .FirstOrNotFoundAsync(r => r.Id == releaseId)
```

Comment:
Yes we've got a comment about it on the backend style guide board to prefer "OrNotFound" extensions and phase out usage of `PersistenceHelper` because it adds no extra value yet requires setting up an extra dependency.